pkg/kubelet/container/container_gc.go

```diff
@@ -39,7 +39,7 @@ type ContainerGCPolicy struct {
 // Implementation is thread-compatible.
 type ContainerGC interface {
 	// Garbage collect containers.
-	GarbageCollect() error
+	GarbageCollect(allSourcesReady bool) error
```

Comment:
Using boolean parameters in a public method is generally frowned upon. Just my 2 cents.